core-bundle/src/Image/ImageSizes.php

```diff
@@ -119,6 +139,19 @@ private function loadOptions(): void
 
         $options = [];
 
+        foreach ($this->predefinedSizes as $name => $imageSize) {
+            if (!isset($options['image_sizes'])) {
+                $options['image_sizes'] = [];
+            }
+
+            $options['image_sizes'][$name] = sprintf(
+                '%s (%sx%s)',
+                $this->translator->trans('IMAGE_SIZES.' . substr($name, 1), [], 'contao_default') ?: substr($name, 1),
```

Comment:
What exactly does this do? There is no `$GLOBALS['TL_LANG']['IMAGE_SIZES']` array.

Reply:
You are right there is none at the moment because we don't have any predefined sizes. However I would like to have this optional array as a place for labels for those predefined image sizes.

Reply:
Not sure about this. I would prefer to define the label (or translation key) in the config file.

Reply:
But then you can't define multilingual labels, or can you?

Reply:
If you provide a translation key, the Symfony translator will find the corresponding label. This implies to create a language file following the Symfony standard of course.